File: python/problem-1017.py
# https://www.beecrowd.com.br/judge/en/problems/view/1017

spent_time = int(input())
average_speed = int(input())

distance = average_speed * spent_time
spent_fuel_liters = distance / 12

print(f"{spent_fuel_liters:.3f}")

File: js/problem-1017.js
const input = require("fs").readFileSync("/dev/stdin", "utf-8");
const [spentTime, averageSpeed] = input.split("\n").map(inp => Number(inp));

const distance = spentTime * averageSpeed;
const spentLiters = distance / 12;
console.log(spentLiters.toFixed(3));